### PR TITLE
inspect comment class differently

### PR DESCRIPTION
**Pull request by yeago**

### CODE REVIEW

File: django_comments/__init__.py [hunk 1]
```diff
@@ -40,13 +40,13 @@ def get_comment_app_name():
...
 
 def get_model():
-    from django_comments.models import Comment
     """
     Returns the comment model class.
     """
     if get_comment_app_name() != DEFAULT_COMMENTS_APP and hasattr(get_comment_app(), "get_model"):
         return get_comment_app().get_model()
     else:
+        from django_comments.models import Comment
```
Comment (yeago) [inline django_comments/__init__.py]: Without this change, I still get http://dpaste.com/1DXR1R4

**Comment by claudep** [comment]
Thanks for the patch

**Comment by yeago** [comment]
@claudep thanks, can i get a tag? :)

**Comment by claudep** [comment]
Not before a new release. I'd like to wait one or two weeks before making a new release.

**Comment by julen** [comment]
Hi @claudep, seeing that a few weeks passed since this was merged, I was wondering if there are any plans for a new release anytime soon. Thank you!

**Comment by claudep** [comment]
Your wishes are orders :-)
https://pypi.python.org/pypi/django-contrib-comments

**Pull request merged**